docs/source/kale.embed.rst

```diff
@@ -49,6 +49,24 @@ kale.embed.image\_cnn module
    :undoc-members:
    :show-inheritance:
 
+kale.embed.lenet module
+------------------------------------
+
+.. automodule:: kale.embed.lenet
+   :members:
+   :undoc-members:
+   :show-inheritance:
+
+
+kale.embed.multimodal\_fusion module
```

Comment:
Are these all feature fusion methods? If yes, call it feature fusion will be better since there are also decision fusion. Feature fusion may be better in kale.embed while decision fusion to be in predict.

Reply:
Changed accordingly